### ISSUE TITLE
registry001 on 2.6.2.0 fails on GHC CI due to compile error

### ISSUE THREAD

**Issue by wz1000** [haskell/win32#181]
The definition of `x` is ambiguous due to the extra exports.

Patch:

```diff
diff --git a/tests/registry001.hs b/tests/registry001.hs
index ade0eba..27a3cb9 100644
--- a/tests/registry001.hs
+++ b/tests/registry001.hs
@@ -1,4 +1,4 @@
-import System.Win32
+import System.Win32 hiding ( COORD(..) )
 import Control.Exception
 import Control.Monad
```

It would be nice if this could be backported to `2.6.2.1` so we don't ship a version of `Win32` with GHC 8.10.6 which fails to pass the testsuite. It is not exactly critical since it should have little user facing impact, but it would be nice to have (very soon if at all possible since 8.10.6 is very close to ready).

This was noticed when I bumped `Win32` for https://gitlab.haskell.org/ghc/ghc/-/issues/20202

I do wish we had a better story for GHC tests which are really a part of submodules. This has come up twice in the last few weeks now.

**Comment by Mistuke** [comment]
I won't be able to do it for that same package version but can release a new one with the fix.

> I do wish we had a better story for GHC tests which are really a part of submodules. This has come up twice in the last few weeks now.

There is a ticket on gitlab about decoupling the ghc testsuite from GHC. That shouldn't be too hard as the testsuite just reads a few config files. But no one has done so yet.

I'll make the change and release tonight.

Thanks!

**Comment by wz1000** [comment]
> I won't be able to do it for that same package version but can release a new one with the fix.

Yes. The version with the problem is 2.6.2.0, I suggested a bump to the last component since this wouldn't affect the user facing parts of the library.

Thinking about it more, it might just be best to backport the fix for #157, which would necessitate a major version bump. I'm not sure how much code will break due to #157 so I leave it up to your judgement.

Thanks!

**Comment by Mistuke** [comment]
Merged via 8103be38865de160fc1ef9e69e84ad87d2556e77 and release trigger for `v2.6,2,1` is running and should be on hackage in 30mins.
